keras_rcnn/layers/losses.py

```diff
@@ -0,0 +1,71 @@
+import keras.layers
+
+
+class Classification(keras.layers.Layer):
+    def __init__(self, anchors, **kwargs):
+        self.anchors = anchors
+
+        self.is_placeholder = True
+
+        super(Classification, self).__init__(**kwargs)
+
+    def _loss(self, y_true, y_pred):
+        # Binary classification loss
+        x, y = y_pred[:, :, :, :], y_true[:, :, :, self.anchors:]
+
+        a = y_true[:, :, :, :self.anchors] * keras.backend.binary_crossentropy(x, y)
+        a = keras.backend.sum(a)
+
+        # Divided by anchor overlaps
+        b = keras.backend.epsilon() + y_true[:, :, :, :self.anchors]
+        b = keras.backend.sum(b)
+
+        return 1.0 * (a / b)
```

Comment:
Why not return `keras.backend.mean(keras.backend.binary_crossentropy(x, y))` directly? Why divide by anchor overlaps?